Harness-engineering-google-guide.html

```
@@ -382,9 +380,9 @@
     </style>
   </head>
   <body>
-    <button type="button" class="nav-toggle" id="navToggle" aria-label="目次を開く" aria-expanded="false" aria-controls="sideNav">☰</button>
+    <button class="nav-toggle" id="navToggle" aria-label="目次を開く">☰</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)